README.md

```diff
@@ -6,9 +6,8 @@ Bundler for non-Ruby dependencies from Homebrew.
 
 ## Requirements
 
-[Homebrew](https://github.com/Homebrew/brew) or [Linuxbrew](https://github.com/Linuxbrew/brew) are used for installing the dependencies.
-Linuxbrew is a fork of Homebrew for Linux, while Homebrew only works on macOS.
-This tool is developed for use with Homebrew on macOS but might work with Linuxbrew (but is unsupported so don't file Linux issues, please).
+[Homebrew](https://github.com/Homebrew/brew) on macOS for installing ependencies.
```

Comment:
```suggestion
[Homebrew](https://github.com/Homebrew/brew) on MacOS for installing the dependencies.
```

Reply:
Thanks, @issyl0. It's macOS, not MacOS. Could you please update the suggestion?

Reply:
Oops, my bad!